sdk/batch/Microsoft.Azure.Batch/Tools/ObjectModelCodeGenerator/Program.cs

```diff
@@ -70,7 +70,7 @@ private static string GetSourceDirectory([System.Runtime.CompilerServices.Caller
 
         private static void GenerateSomeRoslynFiles()
         {
-            var projectFile = @"..\..\..\Azure.Batch\Azure.Batch.csproj";
+            var projectFile = @"..\..\..\Microsoft.Azure.Batch\src\Azure.Batch.csproj";
```

Comment:
Should this be `Microsoft.Azure.Batch.csproj`

Reply:
Yes it should.